conf/gunicorn-dev.conf.py

```diff
@@ -0,0 +1,5 @@
+bind = "0.0.0.0:9082"
+reload = True
+reload_extra_files = "via/templates"
+timeout = 0
+workers = 2
```

Comment:
Similar to the change made in Checkmate: `conf/gunicorn/dev.conf.py` has been moved to `conf/gunicorn-dev.conf.py`. `reload_extra_files` has been added and `workers` has been changed from `4` to `2`, but otherwise the config should be the same.